### ISSUE TITLE
Provide ARN attribute on Glue Crawler

### ISSUE THREAD

**Issue by danielcompton** [terraform-providers/terraform-provider-aws#7934]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I'm trying to create IAM policies to allow processes to start and describe Glue Crawlers. To scope this tightly, I would like to use the ARN of the crawler, rather than just `*`. The `aws_glue_crawler` resource doesn't provide this attribute, it only provides the id (name) of the resource.

### New or Affected Resource(s)

* aws_glue_crawler

### Potential Terraform Configuration

```hcl
${aws_glue_crawler.my_crawler.arn}"
```

### Current Terraform Configuration

Currently, this is how I'm building the ARN:

```hcl
data "aws_region" "current" {}

data "aws_caller_identity" "current" {}

"arn:aws:glue:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:crawler/${aws_glue_crawler.my_crawler.name}"
```

**Comment by bflad** [comment]
Support for the `arn` attribute for the `aws_glue_crawler` resource has been merged and will release with version 2.2.0 of the Terraform AWS Provider shortly.

**Comment by bflad** [comment]
This has been released in [version 2.2.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#220-march-15-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
